File: Sportlance.WebUI/src/app/core/error-code.ts
export enum ErrorCode {
  // Auth
  AuthenticationError = 'AuthenticationError',
  IncorrectPassword = 'IncorrectPassword',
  EmailIsNotConfirmed = 'EmailIsNotConfirmed',
  RegistrationIsAlreadyConfirmed = 'RegistrationIsAlreadyConfirmed',
  // Team
  TeamNotFound = 'TeamNotFound',
  TeamServiceNotFound = 'TeamServiceNotFound',
  DurationIsRequired = 'DurationIsRequired',
  PriceMustBeGreaterThanZero = 'PriceMustBeGreaterThanZero',
  // Trainer
  TrainerNotFound = 'TrainerNotFound',
  // User
  UserNotFound = 'UserNotFound',
  UserAlreadyHasRole = 'UserAlreadyHasRole',
  UserAlreadyExist = 'UserAlreadyExist',
  RoleNotFound = 'RoleNotFound',
  UserAccessDenied = 'UserAccessDenied',
  // Common
  NameIsRequired = 'NameIsRequired',
  // Other
  IncorrectData = 'IncorrectData',
  IncorrectValidation = 'IncorrectValidation',
  ServerError = 'ServerError',
  UpdateInProgress = 'UpdateInProgress',
  TxIsInvalid = 'TxIsInvalid'
}

File: Sportlance.Common/Errors/ErrorCode.cs
﻿namespace Sportlance.Common.Errors
{
    public enum ErrorCode
    {
        // Auth
        AuthenticationError,
        IncorrectPassword,
        EmailIsNotConfirmed,
        RegistrationIsAlreadyConfirmed,
        // Team
        TeamNotFound,
        TeamServiceNotFound,
        TeamServiceAlreadyDeleted,
        DurationIsRequired,
        PriceMustBeGreaterThanZero,
        // Trainer
        TrainerNotFound,
        // User
        UserNotFound,
        UserAlreadyHasRole,
        UserAlreadyExist,
        RoleNotFound,
        UserAccessDenied,
        // Orders
        OrderNotFound,
        // Common
        NameIsRequired,
        // Other
        IncorrectData,
        IncorrectValidation,
        ServerError,
        UpdateInProgress,
        TxIsInvalid
    }
}